build.sc

```diff
@@ -4,15 +4,13 @@ import scalalib._
 import ammonite.ops._
 import mill.scalalib.publish.{Developer, License, PomSettings, VersionControl}
 import $ivy.`com.lihaoyi::mill-contrib-buildinfo:0.4.1`
-import mill.contrib.scoverage.ScoverageModule
+import coursier.maven.MavenRepository
 
-import $file.decco.build
-
-object scalanet extends ScoverageModule {
+object scalanet extends ScalaModule {
 
   def scalaVersion = "2.12.7"
```

Comment:
See couple of duplicates in both scalanet and test objects. Maybe it would be nice to have a common object here with these variables?